packages/protocol/migrations/02_registry.ts

```diff
@@ -1,15 +1,31 @@
-import { deployImplementationAndRepointProxy } from '@celo/protocol/lib/web3-utils'
-import { config } from '@celo/protocol/migrationsConfig'
+import { build_directory, config } from '@celo/protocol/migrationsConfig'
 import { RegistryInstance } from 'types'
+import { setInitialProxyImplementation } from '../lib/web3-utils'
 
-const initializeArgs = async (): Promise<any[]> => {
-  return []
-}
+const Artifactor = require('truffle-artifactor')
+
+const name = 'Registry'
+const Contract = artifacts.require(name)
+const ContractProxy = artifacts.require(name + 'Proxy')
 
-module.exports = deployImplementationAndRepointProxy<RegistryInstance>(
-  web3,
-  artifacts,
-  config.registry.predeployedProxyAddress,
-  'Registry',
-  initializeArgs
-)
+module.exports = (deployer: any, _networkName: string, _accounts: string[]) => {
+  // tslint:disable-next-line: no-console
+  console.log('Deploying Registry')
```

Comment:
Could some of the code from `web3-utils` be abstracted out into a helper function used there and here to deduplicate this proxied deployment logic?

Reply:
decide to keep it as it is; since there is not much logic to extract